README.md

```diff
@@ -1 +1,57 @@
-This is a starter template for [Learn Next.js](https://nextjs.org/learn).
\ No newline at end of file
+# 📜 The Introduction
+
+**Open Source Community Kampala** is an organisation for open-source lovers, enthusiasts, advocates, and experts within and across Uganda but centered in Kampala with the sole aim of increasing the rate of credible contributions by Ugandan software developers, designers, writers, and everyone involved in the sphere of technology to open source projects both locally and globally, changing the perception the world has of our developers; we are not just users of software tools, we are also **Software creators**.
+
+![](.gitbook/assets/osca-logo_colored.png)
```

Comment:
Did we add the osca logo here?

Reply:
I actually foresaw that and removed it; I have made a commit; kindly check and reconcile the facts.